docs/_articles/en/user-guide/byotemplate.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Custom Templates
+lang: en
+---
+
+## Overview
+Want your own code to appear in source files when you create a metadata object such as an Apex class? You can now use custom templates to do just that.
+
+Templates are essentially folders with files that contain your custom code. This [git repo](https://github.com/forcedotcom/salesforcedx-templates/tree/develop/packages/templates/src/templates) contains a collection of official Salesforce templates for metadata components. Simply clone this repo, keeping the same folder structure, then update relevant template files with your code. Remove the files that you don’t wish to override.
+
+**Note:** Only updates made to the files listed here show up in source files in VS Code. There’s no such restriction when you use CLI to specify templates for metadata objects.
+
+| Template Folder        | Default Template Files           |
+| ------------- |:-------------:|
+| apexclass     | DefaultApexClass.cls <br>_class.cls-meta.xml
```

Comment:
Does the markdown converter ignore different spacing within table cells. Noticed sometimes there's a space before or after and sometimes not.

Reply:
It does...I tried a bunch of different things with spaces but I cannot make spaces consistent because I have to add html tags here..

Reply:
I will give it another try and push a separate update if that's ok?